flytekit/core/tracker.py

```diff
@@ -64,6 +64,8 @@ def _find_instance_module():
             if frame.f_code.co_name == "<module>" and "__name__" in frame.f_globals:
                 if frame.f_globals["__name__"] != "__main__":
                     return frame.f_globals["__name__"], frame.f_globals["__file__"]
+                if frame.f_globals["__file__"].endswith("pyflyte"):
```

Comment:
let's create a tuple of the executables we want to exclude.

Reply:
I just removed the changes from `tracker.py`. Now, This PR only adds the tests for the agent.